distributedsocialnetwork/api/views.py

```diff
@@ -397,6 +397,28 @@ def post(self, request, pk):
                 # We insert a comment to this post's comments
                 comment = request.data["comment"]
                 # Our comment model has an author field that is just an ID. So we have to strip that out
+                if len(Node.objects.filter(hostname__contains=comment["author"]["host"])) != 1:
+                    return Response({
+                        "query": "addComment",
+                        "success": False,
+                        "message": "This node is not authorized to post comments on this server."
+                    }, status=status.HTTP_403_FORBIDDEN)
+                elif (len(Author.objects.filter(id=comment["author"]["id"])) == 0):
+                    # Node is authorized but auther does now exit yet
+                    node = list(Node.objects.filter(
+                        hostname__contains=comment["author"]["host"]))[0]
+                    comment["author"]['displayName'] = comment["author"]['displayName'] + \
+                        " (" + node.server_username + ")"
```

Comment:
Don't forget to set the URL appropriately. As of now if you make a new user his link will not work.

Reply:
Sounds good, my eyes were hurting so bad from staring at the screen that I could barely figure out what was going on anymore :')

Reply:
Added change, please review.